### DISCUSSION TITLE
Github Campus Expert Application Eligibility

### DISCUSSION THREAD

**Discussion by czarinacuarez** [community/community#133066] [GitHub Education]
### Hi everyone,

I'm currently an upcoming final year student, our classes will start on August 2024, while my graduation date will be set on August 2025. I wanted to ask if i'm still eligible to apply for the Github Campus Experts Program? Thank you so much.

**Comment by davevad93** [comment]
Hi @czarinacuarez ,

These are the requirements to be eligible for `GitHub Campus Expert`:

- Be a GitHub user for at least six months.
- Be at least 18 years of age.
- Be enrolled in a formal higher education institution.
- Have more than one year left as a student before graduating.
- Not be enrolled in the GitHub Campus Advisors Program.
- Validate your student status through the [GitHub Student Developer Pack](https://education.github.com/pack).

Considering that you only have one year left to graduate, I don't know if your application will be accepted or not, but try anyway.

For more info see the "[Applying to be a GitHub Campus Expert](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert)" documentation.
